dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNDataInterfaceIIDM.cpp

```diff
@@ -173,7 +173,7 @@ DataInterfaceIIDM::getNetworkIIDM() const {
 
 std::string
 DataInterfaceIIDM::getBusName(const std::string& componentName, const std::string& labelNode) {
-  boost::unordered_map<string, shared_ptr<ComponentInterface> >::const_iterator iter = components_.find(componentName);
+  std::map<string, shared_ptr<ComponentInterface> >::const_iterator iter = components_.find(componentName);
```

Comment:
Since this is only to fix non determinism of lostEquipment I would propose to sort the lostEquipment result instead of globally loosing performance with those finds in O(Nlog(N))